Jenkinsfile

```diff
@@ -79,17 +79,15 @@ pipeline {
                 }
             }
         }
-        // The full build will be run against all regular branches
-        stage('Full Build') {
+        stage('Full Build Except Itests') {
             when { expression { env.CHANGE_ID == null } }
             parallel {
                 stage ('Linux') {
                     steps {
-                        timeout(time: 3, unit: 'HOURS') {
+                        timeout(time: 2, unit: 'HOURS') {
```

Comment:
❓ Can we document why we're making this change (down from 3 to 2) somewhere? Not sure if a comment makes the most sense

Reply:
I can't think of a better place for it, so I will put it it in the file for now.